tests/integration/dummy_mqs.py

```diff
@@ -0,0 +1,42 @@
+"""A dummy MQS server for testing."""
+
+
+import os
+import time
+
+from flask import Flask, jsonify, request
+
+app = Flask(__name__)
+
+
+@app.route("/mq-group", methods=["POST"])
+def dummy_mq_group_post():
+    mqgroup_id = "test-mq-group"
+    now = int(time.time())
+
+    return jsonify(
+        dict(
```

Comment:
Question re style: is it better to use `dict()` or `{ }`?  Especially here where you're writing json.

(I feel like I've asked this before, but forgot the response)

Reply:
I've started using `dict()` more since it takes up fewer characters. I think it comes down to preference. One thing (good or bad) it does do is make all keys also valid identifiers (ie no dashes)